doc/nims.rst

```diff
@@ -161,3 +163,5 @@ ends with ``.nims``:
   mode = ScriptMode.Silent
 
   echo "hello world"
+
+Use ``#!/usr/bin/env -S nim --hints:off`` to disable hints.
```

Comment:
Not everybody is on Unix...

Reply:
@Araq your comment also applies to the use of the shebang and env few lines above but it was already in place.
I can replace 'You can also use the shebang' with 'On Posix you can use ...'